ntoskrnl/fsrtl/fastio.c

```diff
@@ -320,6 +320,10 @@ FsRtlCopyWrite(IN PFILE_OBJECT FileObject,
 
     FsRtlEnterFileSystem();
 
+    /* Fail by default */
+    IoStatus->Status = STATUS_UNSUCCESSFUL;
+    IoStatus->Information = 0;
+
```

Comment:
I'm really not keen on that change. Could you do the FsRtlCopyWrite2() way?
You first set success, and only set status on error.
See: sdk/lib/drivers/copysup/